src/vaadin-menu-bar-interactions-mixin.html

```diff
@@ -234,7 +224,7 @@
 
     /** @private */
     __alignOverlayPosition(e) {
-      /* istanbul ignore if */
+      /* c8 ignore next */
```

Comment:
Please revert this change, it was needed because `@web/test-runner` is using a different tool for coverage.

Reply:
Done.